cmd/osbuild-image-tests/main_test.go

```diff
@@ -351,17 +352,20 @@ func testImage(t *testing.T, testcase testcaseStruct, imagePath string) {
 // runTestcase builds the pipeline specified in the testcase and then it
 // tests the result
 func runTestcase(t *testing.T, testcase testcaseStruct) {
-	outputDirectory, err := ioutil.TempDir("/var/tmp", "osbuild-image-tests-*")
-	require.NoErrorf(t, err, "cannot create temporary output directory: %#v", err)
+	tmpdir, err := ioutil.TempDir("/var/tmp", "osbuild-image-tests-*")
```

Comment:
This will slow down test runs, could we use a fixed store?

Reply:
Good point. Will fix.